skxray/roi.py

```diff
@@ -229,3 +231,96 @@ def ring_edges(inner_radius, width, spacing=0, num_rings=None):
     edges = np.cumsum(steps).reshape(-1, 2)
 
     return edges
+
+
+def pie_slices(edges, slicing, center, shape, theta=0):
```

Comment:
@danielballan I added a pie_slices function and it draw the mask we required. I will add some tests to it